github-385-pr: Add Ansible Rulebook to glossary (#386)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/p.adoc

*See also*:

[[playbook]]
==== image:images/yes.png[yes] playbook (noun)

> *Description*: Playbooks are Ansible's configuration, deployment, and orchestration language.

Reviewer: ```suggestion
*Description*: Playbooks are the configuration, deployment, and orchestration language for Ansible Automation Platform.
```
- Per ISG, "Do not use 's to show the possessive form of brand names or product names" - https://www.ibm.com/docs/en/ibm-style?topic=references-product-service-names#general-guidelines"

> Playbooks can describe a policy you want your remote systems to enforce, or a set of steps in a general IT process.

Reviewer: ```suggestion
Playbooks can describe a policy you want your remote systems to enforce or a set of steps in a general IT process.
```
- Remove comma after subordinating conjunction `or`. A comma would only be necessary if  an independent clause followed `or`.

Author: Another copy over from "Ansible Playbook". I'll update that entry also.

When using the term _playbook_ without the Ansible prefix, use lowercase _p_.

Examples:

> * Run a playbook in Ansible.

Reviewer: ```suggestion
* Run a playbook in Ansible.
* Run an Ansible Playbook.
```
- Consider adding an additional example so that you show both acceptable usages.

*See also*:

[[playbook]]
==== image:images/yes.png[yes] playbook (noun)

> *Description*: Playbooks are the configuration, deployment, and orchestration language for Ansible Automation Platform.

Reviewer 2: ```suggestion
*Description*: _Playbooks_ are the configuration, deployment, and orchestration language for Ansible Automation Platform.
```

Playbooks can describe a policy you want your remote systems to enforce or a set of steps in a general IT process.

> When using the term _playbook_ without the Ansible prefix, use lowercase _p_.

Reviewer 3: ```suggestion
When using the term "playbook" without the Ansible prefix, use lowercase _p_.
```
